packages/ui/src/ui/pages/tablet_cell_bundles/bundles/BundlesTable.tsx

```diff
@@ -514,6 +517,7 @@ export type ReduxProps = {
     columns: Array<keyof typeof Columns>;
     activeBundleLink(cluster: string, bundle: string, enable_bundle_controller?: boolean): string;
     bundleDashboardUrl?(cluster: string, bundle: string): string | undefined;
+    writeableByName?: {get: (bundleName: string) => boolean | undefined};
```

Comment:
Is it possible to define field type as `Map<string, boolean>`?

Reply:
Yes, it is allowed to pass instances of Map<string, boolean> as a value for the property. It will be the same if we rewrite the type as 
```ts
writeableByName?: Pick<Map<string, boolean>, 'get'>;
```
If you think it is more clear I will replace the declaration.

Reply:
both variants are ok for me, I just was worried about picking only one method instead of full Map
but then i`ve understood the reason 👍🏻